opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/SimpleStixObjectOrStixRelationshipStixCoreRelationshipLine.jsx

```diff
@@ -433,6 +436,45 @@ const SimpleStixObjectOrStixRelationshipStixCoreRelationshipLineFragment = creat
             }
             ... on StixCoreRelationship {
               relationship_type
+              from {
+                ... on BasicObject {
+                  id
+                  entity_type
```

Comment:
entity_type is probable not necessary ?
you are not fetching it in `to  { ... onBasicRelationship` for instance.